ops/version.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os.path
+import subprocess
+
+__all__ = ('version',)
+
+
+def _get_version():
+    version = "0.7.dev+UNKNOWN"
```

Comment:
This will be always this version, or we would need to remember updating this everytime we update the other value?

Reply:
We need to remember to bump this after every release.
But ... there is no other value. This is the only one we'd need to update.
Setting it to just 'UNKNOWN' would mean _no_ version number in-repo, but I agree we want something like this. The nice thing is that, as it needs updating _after_ the release, it doesn't block the release.